### PR TITLE
Adds an example for Add-NsxSecurityGroupMember

### PR DESCRIPTION
**Pull request by j33tu**
I have added a simple example showing how to add a bunch of VMs to NsxSecurityGroup using add-NsxSecurityGroup

### CODE REVIEW

**Comment by jitendra12singh** [comment]
this is related to #561

File: module/PowerNSX.psm1 [hunk 1]
```diff
@@ -22973,7 +22973,13 @@ function Add-NsxSecurityGroupMember {
 
     A valid PowerCLI session is required to pass certain types of objects
     supported by the IncludeMember and ExcludeMember parameters.
+    
+    .EXAMPLE
+    $vms = @(Get-VM -Name vmname*) # fetch the VMs to be added to the NSX security group
```
Comment (alagoutte) [inline module/PowerNSX.psm1]: Not need to add comment (#...)

File: module/PowerNSX.psm1 [hunk 2]
```diff
@@ -22973,7 +22973,13 @@ function Add-NsxSecurityGroupMember {
...
+    $sg = Get-NsxSecurityGroup -name "securitygroupname" # fetch the Securty group
+    Add-NsxSecurityGroupMember -SecurityGroup $sg -Member $vms -Verbose
```
Comment (alagoutte) [inline module/PowerNSX.psm1]: remove -Verbose ?

File: module/PowerNSX.psm1 [hunk 3]
```diff
@@ -22973,7 +22973,13 @@ function Add-NsxSecurityGroupMember {
...
 
+    This example shows how to add a bunch of vms to a NSX Security group.
```
Comment (alagoutte) [inline module/PowerNSX.psm1]: Make a better example for exchange Add All VM named "XXX- on security SG-XXX ?

add some other example with other SecurityGroup possibilité ?

File: module/PowerNSX.psm1 (hunk 2 above)
Comment (thexvjitendra) [reply module/PowerNSX.psm1]: @alagoutte sure

File: module/PowerNSX.psm1 (hunk 3 above)
Comment (thexvjitendra) [reply module/PowerNSX.psm1]: @alagoutte  working on them now , was out of  git for some time

Comment (thexvjitendra) [reply module/PowerNSX.psm1]: sure , i will

File: module/PowerNSX.psm1 (hunk 2 above)
Comment (thexvjitendra) [reply module/PowerNSX.psm1]: sure

File: module/PowerNSX.psm1 (hunk 1 above)
Comment (thexvjitendra) [reply module/PowerNSX.psm1]: ok

File: module/PowerNSX.psm1 (hunk 2 above)
Comment (alagoutte) [reply module/PowerNSX.psm1]: always -Verbose

Comment (j33tu) [reply module/PowerNSX.psm1]: working on new examples

Comment (j33tu) [reply module/PowerNSX.psm1]: just forked to my local PC ...

**Pull request closed without merging**